src/applications/disability-benefits/686/config/form.js

```diff
@@ -121,81 +124,74 @@ const formConfig = {
     vaFileNumber
   },
   chapters: {
-    veteranInformation: {
-      title: 'Veteran Information',
+    applicantInformation: {
+      title: 'Applicant Information',
       pages: {
-        veteranInformation: {
-          path: 'veteran-information',
-          title: 'Veteran Information',
+        applicantInformation: {
+          title: 'Applicant information',
+          path: 'applicant-information',
           uiSchema: {
-            veteranFullName: fullNameUI,
-            veteranSocialSecurityNumber: _.merge(ssnUI, {
-              'ui:required': form => !form.veteranVAfileNumber
-            }),
-            veteranVAfileNumber: {
-              'ui:title': 'VA file number (must have this or a Social Security number)',
-              'ui:required': form => !form.veteranSocialSecurityNumber,
-              'ui:help': VAFileNumberDescription,
-              'ui:errorMessages': {
-                pattern: 'Your VA file number must be between 7 to 9 digits'
-              }
-            },
-            'view:relationship': {
-              'ui:title': 'Relationship to Veteran',
-              'ui:widget': 'radio',
-              'ui:options': {
-                labels: relationshipLabels
-              }
-            },
-            'view:applicantInfo': {
-              'ui:title': 'Applicant Information',
-              claimantFullName: _.merge(fullNameUI, {
-                first: {
-                  'ui:required': (formData) => formData['view:relationship'] !== 'veteran'
+            'ui:description': applicantDescription,
+            application: {
+              claimant: {
```

Comment:
I don't think you need to nest these fields in `application` and `claimant` objects. Pre-need does that because there will eventually be multiple applicants on that form, but we don't need that here.

Reply:
k